erpnext_documentation/www/docs/user/manual/en/support/issue.md

```diff
@@ -76,6 +83,14 @@ User can filter the issues based on these fields linked to the Issue:
 * **Opening Time**: When the issue is created or logged the exact time will automatically be posted.
 * **Resolution Date**: When the user resolved the issue, Date and Time will get updated in this field.
 * **Resolution Details**: User can enter the details of the issue, once it is resolved. This is a text field. Also, user can upload the image, enter or create a table.
+* **Resolution Time**: Total time taken to close the ticket (from Issue creation to closing).
+* **User Resolution Time**: Many times a user has to wait for a customer's reply in order to solve some Issue. While measuring User's productivity this wait time shouldn't be taken into consideration. Hence, user resolution time is the total time taken by a user to close the ticket calculated as:
```

Comment:
Hence, user resolution time is the total time taken by a user to close the ticket and can be calculated as: